### PR TITLE
Removes system tracking code from the UI

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
link #2137 

The primary purpose of this PR is to remove the system-tracking directory and all references to it.  The ability to view system tracking in the UI was removed some time ago.

It turns out that we were importing src/shared from inside the system tracking module.  Since that was a) no longer possible and b) not correct to begin with, I moved that import to `app.js` with all the other src root imports.

##### ISSUE TYPE
 - Task

##### COMPONENT NAME
 - UI

### CODE REVIEW

**Comment by wenottingham** [comment]
Apologies, this now has conflicts. Can you rebase and fix the conflicts?

File: awx/ui/client/src/app.js [hunk 1]
```diff
@@ -65,13 +65,13 @@ angular
...
         about.name,
         access.name,
         license.name,
         RestServices.name,
         browserData.name,
         configuration.name,
-        systemTracking.name,
         inventoriesHosts.name,
         inventoryScripts.name,
         credentials.name,
```
Comment (jlmitch5) [inline awx/ui/client/src/app.js:74]: on line 96, I think you can remove the old style 'Utilities'

**Comment by mabashian** [comment]
Take a look at some of the less files in shared/utilities (alerts, hidden, layer, and unbold) and truncated-text.less.  We might be able to remove all of those files when we remove the old system tracking code.

**Review by jlmitch5 — APPROVED** [review]
Took another look at this--looks good!  Nice job cleaning up the way `shared` is included

**Pull request merged**